cmd/admin-handlers.go

```diff
@@ -1172,6 +1172,37 @@ func (a adminAPIHandlers) SitePerfHandler(w http.ResponseWriter, r *http.Request
 	}
 }
 
+// ClientDevNull - everything goes to io.Discard
+// [POST] /minio/admin/v3/speedtest/client/devnull
+func (a adminAPIHandlers) ClientDevNull(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	objectAPI, _ := validateAdminReq(ctx, w, r, iampolicy.HealthInfoAdminAction)
```

Comment:
We should introduce perhaps another action here since this is easy to abuse. 

Also have an upper limit of 100G data transfer and also have a global atomic boolean to reject if two requests are sent to same server of ClientDevNull. 

We only allow 1 at a time.